snuba/migrations/migration.py

```diff
@@ -71,7 +71,7 @@ def forwards(self, context: Context, dry_run: bool) -> None:
         update_status(Status.IN_PROGRESS)
 
         for op in self.forwards_global():
-            op.execute()
+            op.execute(logger)
```

Comment:
Why do you need to pass the logger through everywhere instead configuring it in the script and just importing it as we do in the snuba api (https://github.com/getsentry/snuba/blob/41d7fe76aaf8a594e8f6e84015607dcde3f67ad4/snuba/snapshots/postgres_snapshot.py#L96)

Reply:
Planning ahead a bit for when the migrations CLI and framework will not be part of Snuba but the actual migrations will be. We'll want the Snuba migration to use the logger from the migrations library.... figured I'd just implement it that way now even if that end state may be a little while off.

Reply:
cool. makes sense